contrib/epee/src/abstract_http_client.cpp

```diff
@@ -36,7 +35,7 @@ namespace net_utils
 
     while (num_char >= radix)
     {
-      temp = num_char % radix;
+      const int temp = num_char % radix;
```

Comment:
Would suggest just inlining this in the array indexing operation, rather than having a single-use variable named `temp`.

Reply:
Note: The above comment requires moving the `csTmp` assign above the `num_char` assign.